shell-completion/zsh/_timedatectl

```diff
@@ -36,6 +36,8 @@ _timedatectl_command(){
         'set-time:Set system time'
         'set-timezone:Set system timezone'
         'list-timezones:Show known timezones'
+        'timesync-status:Show status of systemd-timesyncd'
+	    'show-timesync:Show properties of systemd-timesyncd'
```

Comment:
Spurious indentation.

Reply:
Hmm, my neovim inserted them as tab originally, tried to fix it but still not correct as it seems. Just a moment, I will try to fix it

Reply:
Should be fixed now